syntax = "proto2";
import "game_commands.proto";
message Command_AttachCard {
    extend GameCommand {
        optional Command_AttachCard ext = 1009;
    }
    optional string start_zone = 1;
    optional sint32 card_id = 2 [default = -1];
    optional sint32 target_player_id = 3 [default = -1];
    optional string target_zone = 4;
    optional sint32 target_card_id = 5 [default = -1];
}


